VideoAPI/Video.API/Controllers/ConferenceManagementController.cs

```diff
@@ -97,5 +98,45 @@ public async Task<IActionResult> EndVideoHearingAsync(Guid conferenceId)
                 return StatusCode(ex.StatusCode, ex.Response);
             }
         }
+        
+        /// <summary>
+        /// Transfer a participant in or out of a hearing
+        /// </summary>
+        /// <param name="conferenceId">Id for conference</param>
+        /// <param name="transferRequest">Participant and direction of transfer</param>
+        /// <returns></returns>
+        [HttpPost("{conferenceId}/transfer")]
+        [SwaggerOperation(OperationId = "TransferParticipant")]
+        [ProducesResponseType((int) HttpStatusCode.OK)]
+        public async Task<IActionResult> TransferParticipantAsync(Guid conferenceId, TransferParticipantRequest transferRequest)
+        {
+            _logger.LogDebug("TransferParticipant");
+            try
+            {
+                switch (transferRequest.TransferType)
+                {
+                    case TransferType.Call:
+                        _logger.LogDebug($"Transferring participant {transferRequest.ParticipantId} into hearing room");
+                        await _videoPlatformService.TransferParticipantAsync(conferenceId,
+                            transferRequest.ParticipantId,
+                            RoomType.WaitingRoom, RoomType.HearingRoom);
+                        break;
+                    case TransferType.Dismiss:
+                        _logger.LogDebug(
+                            $"Transferring participant {transferRequest.ParticipantId} out of hearing room");
+                        await _videoPlatformService.TransferParticipantAsync(conferenceId,
+                            transferRequest.ParticipantId,
+                            RoomType.HearingRoom, RoomType.WaitingRoom);
+                        break;
+                }
```

Comment:
not sure if youll agree but I always like to add a default block and throw an `InvalidOperationException`, I know currently it's not needed but if another type is added sometime
up to you 👍

Reply:
I've got no objection. Good idea to harden the endpoint and protect ourselves from future unhandled changes